reliability/README.md

```diff
@@ -1,103 +1,41 @@
 # Drivy Reliability Challenge
 
-Looking for a job? Check out our [https://en.drivy.com/jobs](https://en.drivy.com/jobs).
+Looking for a job? Check out our [open positions](https://en.drivy.com/jobs).
+You can also take a look at our [engineering blog](https://drivy.engineering/) to learn more about the way we work.
 
 ## Guidelines
-- Clone this repo (do not fork it)
-- Commit your code
-- Once you are done, ping someone from Drivy (ideally via our jobs page)
 
-Please do the simplest thing that could work for the level you're currently solving.
+- Clone this repo (do **not** fork it)
+- Solve the levels in ascending order
+- Only do one commit per level and include the `.git` when submiting your test
 
-For higher levels we are interested in seeing code that is:
+## Pointers
 
-- Clean
-- Extensible
-- Reliable
+You can have a look at the higher levels, but please do the simplest thing that could work for the level you're currently solving.
 
-## Challenge
+The levels become more complex over time, so you will probably have to re-use some code and adapt it to the new requirements.
+A good way to solve this is by using OOP, adding new layers of abstraction when they become necessary and possibly write tests so you don't break what you have already done.
 
-The challenge needs to be resolved in Ruby.
```

Comment:
This is not required anymore?

Reply:
Yes we would prefer Ruby, but if the candidate think he/she can produce something marginally better in another language we'll take it.
Finding Ruby developers is hard enough, we don't want to close any doors.